modules/client-configuration/pages/client-upgrades.adoc

```diff
@@ -1,16 +1,21 @@
 [[client-upgrades]]
 = Client Upgrades
 
+ifeval::[{suma-content} == true]
 Clients use the versioning system of their underlying operating system, and require regular upgrades.
```

Comment:
This is also valid for Uyuni. Uyuni can manage SLE.

Reply:
ok, but sb must sort out the details ;)  With this PR I tried to solve an issue a user reported on the uyuni ML (#12527).  Of course, your concerns are very valid but we must track and solve them separately.

I just created https://github.com/SUSE/spacewalk/issues/12795.